ccdproc/combiner.py

```diff
@@ -1,7 +1,6 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
 """This module implements the combiner class."""
-
```

Comment:
Could be kept. I like blank lines between module docs and imports.

Reply:
Oops, that was unintentional...